### ISSUE TITLE
[Test] TestRedisService.HugeCommandInline might produce a huge output

### ISSUE THREAD

**Issue by frozenspider** [yugabyte/yugabyte-db#10893]
### Description

When running test `TestRedisService.HugeCommandInline`, it can result in
```
../../src/yb/yql/redis/redisserver/redisserver-test.cc:948
       Expected: reply_type
       Which is: kError
       To be equal to: reply.get_type()
       Which is: kArray
 Originator: ../../src/yb/yql/redis/redisserver/redisserver-test.cc:1089, reply: {kArray, [{kString, subkey1}, {kString, TTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTT
```
with 277 MB of `T`s. We don't really want to log a string that long because it breaks our tools.

**Comment by frozenspider** [comment]
Fixed by b0d9176

**Comment by mbautin** [comment]
Thanks @d-uspenskiy @frozenspider! It would be good to backport this to branches as well.
